integration/ui/connect_chatplugin.spec.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  cyChannelsChatPluginAddButton,
+  cyChannelsChatPluginConnectButton,
+  cyChannelsChatPluginFormNameInput,
+  cyChannelsChatPluginFormSubmitButton,
+  cyChannelsChatPluginFormBackButton,
+  cyChannelsChatPluginList,
+} from 'handles';
+
+describe('Connect chatplugin channel', () => {
+  it('Logs into the UI', () => {
+    const chatPluginName = 'Chatplugin';
```

Comment:
`Cypress Chatplugin` should help us avoid name clashing with other channels.
Naming is hard *sigh*